presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -268,4 +354,18 @@ private String getResourcePath(String resourceName)
     {
         return this.getClass().getClassLoader().getResource(resourceName).getPath();
     }
+
+    private static void assertDenied(ThrowingCallable callable, String expectedErrMessage)
+    {
+        assertThatThrownBy(callable)
+                .isInstanceOf(AccessDeniedException.class)
+                .hasMessageStartingWith(expectedErrMessage);
```

Comment:
check that message equals or matches the pattern

Reply:
@kokosing - I have changed to hasMessageMatching() and using pattern to match. Let me know if that's ok.